internal/service/resource_pool_fetcher.go

```diff
@@ -192,33 +160,29 @@ func (b *ResourcePoolFetcherBuilder) Build() (
 		extensions:    b.extensions,
 		graphqlQuery:  b.graphqlQuery,
 		graphqlVars:   b.graphqlVars,
-		jqTool:        jqTool,
 	}
 	return
 }
 
 // FetchItems returns a data stream of O2 ResourcePools.
 // The items are converted from Clusters fetched from the search API.
 func (r *ResourcePoolFetcher) FetchItems(
-	ctx context.Context) (resourcePools data.Stream, err error) {
+	ctx context.Context) (clusters data.Stream, err error) {
 	// Search Clusters
 	response, err := r.getSearchResponse(ctx)
 	if err != nil {
 		return
 	}
 
 	// Create reader for Clusters
-	clusters, err := k8s.NewStream().
+	clusters, err = k8s.NewStream().
 		SetLogger(r.logger).
 		SetReader(response).
 		Build()
 	if err != nil {
 		return
 	}
 
-	// Transform Clusters to ResourcePools
```

Comment:
why deleting this and adding them outside? You are changing the whole purpose of this method.

Reply:
I am changing this because it was done differently in the resource pool fetcher vs the resource fetcher.  The resource fetcher did a good job of separating the concerns of collecting the data (fetcher) and mapping the data to the API model (handler).  The resource pool fetch did not separate those concerns and did both tasks in the same piece of code.  I intend on re-using the fetchers to pull the data into the new resource server persisted storage and the mapping to the old API model is an unnecessary step so I am moving it into the handler where it belongs.   Once we switch over to the new resource server then the handlers will all be deleted.

Reply:
I see, so all this will be deleted then. No need to modify name of method or go docs to align to what it is doing now.